portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,35 +27,38 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> funFacts = new ArrayList<String>();
+  private ArrayList<String> comments = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      funFacts.add("I used to read the dictionary for fun!");
-      funFacts.add("I considered going to the University of Madison (Better Dead Than Red)");
-      funFacts.add("I spent over $100 on a planner in the 10th grade!");
+    response.setContentType("text/html;");
+    response.getWriter().println(comments);
+  }
+
+
```

Comment:
formatting nit: Use 1 space between methods. You can use the linter I shared and that should do all the formatting for you.

Reply:
Done! And I'll be using the linter moving forward